.gitignore

```diff
@@ -14,3 +14,7 @@ __pycache__/
 # Coverage information
 /.coverage
 /htmlcov/
+
+# Vi(m)'s swap files
+*.swp
+*.swo
```

Comment:
These swapfiles are produced because an individual developer chose to use vim, not because the project necessitated the use of vim. Consequently, they are best ignored in the individual developer's Git configuration, not in the project-wide Git configuration.

You can do this like so:

```sh
cat >>~/.gitconfig <<EOF
[core]
    excludesfile = /home/peterlacko/.gitignore_global
EOF
echo '*.swp' > ~/.gitignore_global
```